app/helpers/dashboard_helper.rb

```diff
@@ -4,7 +4,11 @@ def lighten_if_zero(value)
   end
 
   def dash_if_zero(value)
-    (value == nil || value == 0) ? "-" : value
+    is_zero(value) ? "-" : value
+  end
+
+  def is_zero(value)
```

Comment:
```suggestion
  def zero?(value)
```

Reply:
Updated the method name to be more idiomatic.